src/components/inputs/toggle-switch.css

```diff
@@ -0,0 +1,52 @@
+.switch {
```

Comment:
@Eduardclt01 try using the same naming convention as you've done for the other components, e.g. this should probably be: `.toggle-switch` instead of `.switch`